lib/edgehog_device_forwarder/application.ex

```diff
@@ -12,6 +12,7 @@ defmodule EdgehogDeviceForwarder.Application do
       EdgehogDeviceForwarderWeb.Telemetry,
       {Phoenix.PubSub, name: EdgehogDeviceForwarder.PubSub},
       EdgehogDeviceForwarder.Supervisors.TerminationCallbacks,
```

Comment:
See below about the `Supervisors` context, the TerminationCallbacks one slip through too, fix it in a separate PR